sources/packages/backend/apps/api/src/route-controllers/student/student.institutions.controller.ts

```diff
@@ -45,4 +53,18 @@ export class StudentInstitutionsController extends BaseController {
       students,
     );
   }
+
+  /**
+   * Get the student information that represents the profile.
+   * TODO: Authorization must be enabled to validate if the student has submitted
+   * at least one application for the institution of the user.
+   * @param studentId student.
+   * @returns student profile details.
+   */
+  @Get(":studentId")
```

Comment:
`@ApiNotFoundResponse({`?

Reply:
mm, I am not seeing this change

Reply:
I can see the change. 
![image](https://user-images.githubusercontent.com/54600590/231299681-18802d39-8057-4c34-afb3-da5a03c5c7fc.png)